fontforge/scripting.c

```diff
@@ -2374,8 +2375,6 @@ static void bExport(Context *c) {
 	if ( c->curfv->selected[i] && (gid=c->curfv->map->map[i])!=-1 &&
 		SCWorthOutputting(c->curfv->sf->glyphs[gid]) )
 	    ScriptExport(c->curfv->sf,bdf,format,gid,format_spec,c->curfv->map);
```

Comment:
I'm not sure it matters. The whole purpose of `tmp` is to simply store the address of the dynamically allocated buffer, such that it can be freed later.